### DISCUSSION TITLE
How to allow snmp to monitor Security Onion Server

### DISCUSSION THREAD

**Discussion by jgiuliano2021** [Security-Onion-Solutions/securityonion#2957] [Unsupported Versions] [answered]
Running Security Onion 2.3.21 from iso.

I have tried allowing 161 via iptables but the changes don't stick. There is no option in so-allow for snmp (161)

I run Centreon as my monitoring platform. In the older version of Security Onion I just allowed through ufw and all was fine.

Anyone know the solution?

Thanks
Joe

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/firewall.html?highlight=firewall
